tests/syntax/test_specifiers.py

```diff
@@ -230,10 +555,37 @@ def test_in():
         assert 150 <= pos.y <= 250
         assert scene.egoObject.heading == 0
 
+def test_in_3d():
+    scenario = compileScenic(
+        'region = BoxRegion(dimensions=(10,20,30))\n'
+        'ego = new Object in region'
+    )
+    for i in range(30):
+        scene = sampleScene(scenario)
```

Comment:
Might as well use `sampleEgo` and simplify all of these tests where you don't need any information from the scene except the ego.